### PR TITLE
Fixes #7707: Reorganize the documentation content

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7707

### CODE REVIEW

**Comment by amousset** [comment]
WIP

File: 10_install_server/00_install_intro.txt [hunk 1]
```diff
@@ -28,4 +28,3 @@ Slower growth over time.
 Reports logs can easily grow to 1.5GB per day.
 
 ======
-
```
Comment (jooooooon) [inline 10_install_server/00_install_intro.txt]: If I recall correctly, this blank line is necessary for the *old style* documentation to build correctly...

File: 6_handbook/0_handbook_intro.txt [hunk 2]
```diff
@@ -1,4 +0,0 @@
-== Handbook
-
-This chapter contains some tips and tricks you might want to know about using Rudder in a production environment, with some useful optimization and procedures.
-
```
Comment (jooooooon) [inline 6_handbook/0_handbook_intro.txt]: Why remove the intro? I'm not against it, I'd just like to understand the rationale.

File: 10_install_server/00_install_intro.txt (hunk 1 above)
Comment (amousset) [inline 10_install_server/00_install_intro.txt]: Yes, thanks.

File: 6_handbook/0_handbook_intro.txt (hunk 2 above)
Comment (amousset) [inline 6_handbook/0_handbook_intro.txt]: I removed it to integrate the sections into the *Administration* chapter.

**Comment by amousset** [comment]
A part of this has been done in #160, I will make another PR for the rest.

**Pull request closed without merging**